### PR TITLE
Fixed assets path and used double quotes to be consistent with the im…

### LINKED ISSUE #159: Re: 'Import CSS, assets and JavaScript'
Problem with 'Import CSS, assets and JavaScript' (https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/)

The **$govuk-images-path** and **$govuk-fonts-path** correctly specify that you require a trailing slash. 

The **$govuk-assets-path** also requires a trailing slash but the docs do not specify as such.

govuk-assets

### PR DESCRIPTION
**Pull request by bfarrimo**
…ages and fonts paths

Fixes #159

### CODE REVIEW

**Review by 36degrees — APPROVED** [review]
This makes sense to me – thanks @bfarrimo 👍🏻 

Just needs a second review from the team and then we can get this merged.

**Comment by 36degrees** [comment]
@calvin-lau-sig7 or @EoinShaughnessy would either of you be able to review?

**Review by EoinShaughnessy — APPROVED** [review]
Looks good! Thanks @bfarrimo

**Pull request merged**
